stat/distuv/poisson_test.go

```diff
@@ -120,7 +121,7 @@ func TestPoisson(t *testing.T) {
 func testPoisson(t *testing.T, p Poisson, i int) {
 	const (
 		tol = 1e-2
-		n   = 1e6
+		n   = 2e6
```

Comment:
out of interest, why do we increase the sample size?

Reply:
The skewness test fails with a small error on the 1e6 sample. Higher sample sizes pass the test, so I assume it's just sampling error. We can also set the random seed to 2 instead of 1 if test slowdown is a concern.

Reply:
nah, that's fine. I was just being curious.